### PR TITLE
Automated Test - COMMERCE-6193 | master

### PR DESCRIPTION
**Pull request by wesleyroberts**

### CODE REVIEW

**Comment by wesleyroberts** [comment]
https://issues.liferay.com/browse/COMMERCE-6193; cc: @lorenzolapescara

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
Review started.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -256,12 +256,63 @@ definition {
...
+		CommerceAcceleratorsInitializer.inviteNewMemberWithRoleToAccount(
+			emailAddress = "buyer@liferay.com",
+			role = "Buyer");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		Click(
+			key_productName = "U-Joint",
+			locator1 = "CommerceAccelerators#PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD");
+
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:311]: Add an `AssertElementNotPresent` that checks that `5` is not a selectable quantity.
```suggestion

		AssertElementNotPresent(
			key_optionValue = "5",
			key_productName = "U-Joint",
			locator1 = "CommerceAccelerators#PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD_OPTION_VALUE");

```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 2]
```diff
@@ -256,12 +256,63 @@ definition {
...
+			accountName = "Test Test",
+			adminEmailAddress = "test@liferay.com");
+
+		Click(
+			key_navUnderline = "Members",
+			locator1 = "NavUnderline#NAV_UNDERLINE_LINK");
+
+		CommerceAcceleratorsInitializer.inviteNewMemberWithRoleToAccount(
+			emailAddress = "buyer@liferay.com",
+			role = "Buyer");
```
[the remark this answers is not in the document]
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: change made

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 3]
```diff
@@ -256,12 +256,63 @@ definition {
...
+			value1 = "6");
+
+		Type(
+			key_fieldLabel = "Multiple Order Quantity",
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = "5");
+
+		Button.clickPublish();
+
+		Pause(locator1 = "1000");
```
[the remark this answers is not in the document]
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: change made

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 4]
```diff
@@ -256,12 +256,63 @@ definition {
 	}
 
 	@description = "COMMERCE-6193. As a buyer, I want the first selectable quantity of a product to be the minimum multiple quantity if Minimum Order Quantity is higher than Multiple Order Quantity"
-	@ignore = "true"
+
```
[the remark this answers is not in the document]
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: change made

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path [hunk 5]
```diff
@@ -330,6 +330,11 @@
 <tr>
 	<td>SELECT_PRODUCT_RELATIONS</td>
 	<td>//div[contains(text(), '${productName}')]/../..//td//input[@type='checkbox']</td>
+	<td></td>	
+</tr>
+<tr>
+<td>PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD_OPTION_VALUE</td>
+	<td>//p[@title='${key_productName}']//../..//select[(@class='form-control quantity-selector')]//option[@value='${key_optionValue}']</td>
```
[the remark this answers is not in the document]
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path]: change made

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase (hunk 1 above)
Comment (wesleyroberts) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:438]: pending due to a bug related to the COMMERCE-6396 ticket

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Comment by kristinonias** [comment]
Element is not present at "//p[@title='U-Joint']//../..//select[(@class='form-control quantity-selector')]//option[@value='10']"

**Comment by wesleyroberts** [comment]
I changed the patch that makes Assert that the least selectable quantity is "10", before it was PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD_OPTION_VALUE now and PRODUCT_CARD_QUANTITY_SELECTOR_INPUT_FIELD_OPTION_VALUE

**Comment by kristinonias** [comment]
@wesleyroberts 

There is commented code in this pull request. Please fix and resend.

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:commerce-functional

**Pull request closed without merging**